configuration.html

```diff
@@ -23,11 +23,11 @@
     <script src="Scripts/recordFinder.js" type="text/javascript"></script>
 
     <style>
-        .users select {
+        .users select, .entities select {
```

Comment:
Why are these selectors combined? Let's keep them separate and refine the model.

Reply:
Split them out